svc/FetchService.js

```diff
@@ -82,6 +82,10 @@ export class FetchService {
         try {
```

Comment:
Don't think we want to complicate our general fetch service with knowledge about this very specific handling of this one type of fetch. 

The most general solution would be to look to sanitize any param with the name "password" in our exception handling code.  As a stopgap,  we might just have explicit exception handling code in LoginPanel.js, rather than using .catchDefault()